docs/guides/testing.md

```diff
@@ -52,12 +52,14 @@ this means your application logic does not need to be changed or mocked when wri
 
 ## Setting Up Zustand for testing
 
-Since Jest and Vitest have slight differences, like Vitest using **ES modules** and Jest using
+> **Note**: Since Jest and Vitest have slight differences, like Vitest using **ES modules** and Jest using
 **CommonJS modules**, you need to keep that in mind if you are using Vitest instead of Jest.
```

Comment:
You can prefix it with `> `, so that it is clear that this line is also intended to be in a block quote.

Reply:
Thanks! That's a good one to know